internal/pkg/agent/cmd/uninstall.go

```diff
@@ -34,6 +34,7 @@ Unless -f is used this command will ask confirmation before performing removal.
 	}
 
 	cmd.Flags().BoolP("force", "f", false, "Force overwrite the current and do not prompt for confirmation")
+	cmd.Flags().String("uninstall-token", "", "Uninstall protection token")
```

Comment:
I think some more detail on this flags description would be helpful for users running `--help`.

Reply:
maybe "Uninstall token required for uninstall protected agent"

Reply:
"Uninstall token required for protected agent uninstall"